apps/mobile/global.css

```
@@ -2,55 +2,65 @@
+			--color-ring: hsl(0 0% 63%);
+			--color-chart-1: hsl(12 76% 61%);
+			--color-chart-2: hsl(173 58% 39%);
+			--color-chart-3: hsl(197 37% 24%);
+			--color-chart-4: hsl(43 74% 66%);
+			--color-chart-5: hsl(27 87% 67%);
 		}
 
 		@variant dark {
-			--color-radius: 0.5rem;
-			--color-background: hsl(240 10% 3.9%);
+			--color-background: hsl(0 0% 3.9%);
```

Biome (2.4.15)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

(lint/suspicious/noDuplicateCustomProperties)